components/molecules/InsightRow/insight-row.tsx

```diff
@@ -73,16 +73,16 @@ const InsightRow = ({ insight, user }: InsightRowProps) => {
             </div>
           </div>
           <div className="flex-1 hidden md:flex  justify-end">
-            <span className=" bg-light-slate-1 inline-block rounded-lg p-2.5 border mr-2">
+            <span className=" bg-light-slate-1 inline-block rounded-lg p-2.5 border mr-2 cursor-pointer">
```

Comment:
This doesn’t change the click behavior, instead, it give an unclickable pointer when hovered on the span element.

A suggestion is to make the span the link instead of nesting within to achieve the title of this PR 🍕

Reply:
Good Catch!